src/content/development/building-testing/_index.en.md

```diff
@@ -81,6 +81,12 @@ make e2e using=helm,globalnet
 
 The [cable driver](../../getting-started/architecture/gateway-engine) used to connect clusters can also be selected with `using` flags:
 
+```shell
+make e2e using=vxlan
```

Comment:
I worry a bit about providing VXLAN as an example that might be copy/pasted without noting that it shouldn't be used on unencrypted connections. Do we need to have a VXLAN example, or can we just use the OVN example and add the "in order to deploy clusters with OVN-Kubernetes" explanation?

Reply:
Alternatively, we can use wireguard, but IMHO it's fine to mention VXLAN here. In the end, this is only a local build/test environment.

Reply:
And FWIW, we have this pending [issue](https://github.com/submariner-io/submariner-website/issues/597) to better clarify how/when VXLAN should be used.